docs/markdown/Wrap-dependency-system-manual.md

```diff
@@ -147,6 +149,37 @@ wrap-file mode. When using wrap-git, the repository must contain all
 Meson build definitions. Since *0.55.0* Meson build patches are
 supported for any wrap modes, including wrap-git.
 
+## Diff files
+
+*Since: 0.61.0*
+
+You can also provide local patch files in `diff` format. For historic reasons,
+they are referred to as "diff files", since the "patch" name is already used for
+Meson build patches.
+
+The diff files are described by the `diff_files` property (a comma-separated
+list), and must be available locally in the `subprojects/packagefiles`
+directory.
```

Comment:
They are searched for at the top level? If applying more than one, I guess paths with directory components would be used for grouping?

Reply:
>They are searched for at the top level?

Yes. The original PR used the top level, and I think that's a good choice because it's consistent with behavior of `source_filename`, `patch_filename`, and `patch_directory` (which also do not expect any extra subdirectory).

>If applying more than one, I guess paths with directory components would be used for grouping?

Yes, I think so. Actually, I'll edit the example to reflect that.

Reply:
The path is relative to `subprojects/packagefiles` but user can decide to add subdirectories of course.